index.bs

```diff
@@ -1162,6 +1162,7 @@ partial interface HTMLIFrameElement {
     1. Let |report-only result| be the result of calling <a abstract-op>Check
        permissions policy</a>, given |report-only policy|, |feature|, |origin|,
        and |document|'s [=Document/origin=].
+    1. Let |result| be "<code>Disabled</code>" at the user agent's discretion.
```

Comment:
You want "set" instead of "let", since `|result|` has already been brought into existence on L1159. But I would personally write:

```suggestion
    1. Optionally, set |result| to "<code>Disabled</code>" if the user agent deems it necessary.

       Note: The motivation of letting the user agent intervene here is driven by [... your documentation here describing what happens here in practice, but not setting it in stone, hence the Note ...]
```

Reply:
Thanks! Done.